rust/monovertex/src/forwarder.rs

```diff
@@ -40,9 +34,7 @@ impl Forwarder {
                 MONO_VERTEX_NAME.to_string(),
                 config().mono_vertex_name.clone(),
             ),
-            (VERTEX_TYPE_LABEL.to_string(), MONO_VERTEX_TYPE.to_string()),
             (REPLICA_LABEL.to_string(), config().replica.to_string()),
```

Comment:
For the replica label, we are reusing `replica` here but we also defined a separate one for the mono vertex above, which is `mvtx_replica`, should it be consistent?